AWS/EC2/ec2-install.sh

```diff
@@ -18,7 +18,7 @@ trap cleanup SIGINT SIGTERM ERR EXIT
 # --- Variables ---
 INSTALL_K3S_VERSION=${INSTALL_K3S_VERSION:-"v1.32.3+k3s1"}
 DV_CHART_REGISTRY_URL=${DV_CHART_REGISTRY_URL:-"https://helm.gradle.com/"}
-DV_VERSION=${DV_VERSION:-"2025.1.0"}
+DV_VERSION=${DV_VERSION:-"2025.1.1"}
 Script_Version="0.1"
 
```

Comment:
This commented out code should not be here

<img width="1039" alt="image" src="https://github.com/user-attachments/assets/c5c7e7f1-b10f-42f5-812a-a99f294b5ce3" />